lms/templates/static_templates/embedded_new_tab.html

```diff
@@ -11,15 +11,38 @@
             dataType: 'json',
             contentType: 'application/json; charset=utf-8',
             success: function(data) {
+                var isTimeExam = $('#time-exam').val();
                 if (data.cookie_sent) {
-                    window.location.href = window.location.href + $('#embedded-link').val();
+                    if (isTimeExam === '1') {
+                        var isInIframe = inIframe();
+                        if (isInIframe){
+                            showNewTabMsg();
+                        }
+                        else {
+                            window.location.href = window.location.href + $('#embedded-link').val() + '&iframe=false';
+                        }
+                    } else {
+                        window.location.href = window.location.href + $('#embedded-link').val() + '&iframe=' + inIframe();
```

Comment:
Concatenation of string and boolean values may get wrong result. I suggest to use `1` and `0` instead of string `true` and `false`. And modify `inIframe` function also